### PR TITLE
Appium find overloads

### PR DESCRIPTION
**Pull request by ncalaway**
Added overloads for finding by text and using offset and limit on findAll

### CODE REVIEW

**Comment by ncalaway** [comment]
Please feel free to suggest ways to clean this up a little. @kbabcock1

File: src/appium/appiumresponse.ts [hunk 1]
```diff
@@ -54,22 +88,61 @@ export class AppiumResponse extends ProtoResponse implements iResponse {
...
-      `/session/${this.scenario.get("sessionId")}/elements`,
-      {
-        method: "post",
-        data: {
-          using: usingValue[0],
-          value: usingValue[1],
-        },
+    let elements: iValue[] = [];
+    const params = getFindParams(a, b);
+    console.log("params: " + params.opts);
```
Comment (johnsickels) [inline src/appium/appiumresponse.ts:101]: Do we want this log?

File: src/appium/appiumresponse.ts [hunk 2]
```diff
@@ -79,6 +152,11 @@ export class AppiumResponse extends ProtoResponse implements iResponse {
         )
       );
     }
+    if (params.opts) {
+      console.log(params.opts);
```
Comment (johnsickels) [inline src/appium/appiumresponse.ts:156]: Log

File: tests/src/error.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { Flagpole } from "../../dist/index.js";
+import { Flagpole, JsonResponse } from "../../dist/index.js";
```
Comment (johnsickels) [inline tests/src/error.ts:1]: Unnecessary import

**Review by johnsickels — APPROVED** [review]

**Pull request closed without merging**